tests/dummy/app/controllers/browsers.js

```diff
@@ -30,9 +30,6 @@ export default Ember.Controller.extend({
             name: 'drop-button',
             lib: { 'twb': true }
         }, {
-            name: 'drop-option',
```

Comment:
This is not related to the `sl-panel` component.  Is this modification intentional?